src/inc/quic_trace_lttng.h

```diff
@@ -1132,6 +1132,12 @@ QUIC_TRACE_EVENT(DatapathErrorStatus,
         ctf_string(Msg, Msg))
 )
 QUIC_TRACE_LEVEL(DatapathErrorStatus, TRACE_ERR)
+QUIC_TRACE_EVENT(DatapathWorkerThread, 
+    TP_ARGS(const void*, UdpBinding),
+    TP_FIELDS(
+        ctf_integer_hex(uint64_t, UdpBinding, UdpBinding))
+)
+QUIC_TRACE_LEVEL(DatapathWorkerThread, TRACE_INFO)
```

Comment:
This shouldn't be needed. It's only necessary for ETW based events (`QuicTraceEvent`).